FlowCrypt/Functionality/Services/Folders Services/RemoteFoldersProviderType/GmailService+folders.swift

```diff
@@ -79,7 +83,7 @@ private extension Folder {
         self.init(
             path: path,
             name: name,
-            image: nil
+            image: UIImage(systemName: path.mailFolderIcon)?.jpegData(compressionQuality: 1.0)
```

Comment:
I like how menu looks now with all these icons 👍

But I think we shouldn't store rendered iOS icons in database. Instead we can use `mailFolderIcon` if folder `image` field is empty:
```swift
image: object.image.flatMap(UIImage.init) ?? UIImage(systemName: object.name.mailFolderIcon),
```
(https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Functionality/Services/Folders%20Services/Models/FolderViewModel.swift#L43)

@ioanmo226 what do you think?

Reply:
I absolutely agree. Thanks for your notice

Reply:
After Folder model is initialized, it will be saved into Realm (FolderRealmObject) database so that we can use it( without getting folder list again)  after initial fetch

Reply:
I see.